docs/installation/gce-kubernetes.md

```diff
@@ -73,4 +73,4 @@ title: GCE Kubernetes
     kubectl get ingress --namespace web android-generator-notls
     ```
 - Wait for a few mins (5-10m) before you move on to the next step.    
-- The generator will be accessible at the above mentioned IP address.
\ No newline at end of file
+- The generator will be accessible at the above mentioned IP address.
```

Comment:
Still there is a 3rd unneeded file in the commit.

Also, please squash before pushing

And avoid unnecessary changes like reformatting. Format only the lines changed by you so they conform to the global standard